src/DataFrames.jl

```diff
@@ -92,6 +92,8 @@ export @~,
 ##
 ##############################################################################
 
+const displaysize(io::IO) = Compat.displaysize()
```

Comment:
Not really, no. Only that it won't change. It's just what I've seen done in Compat so I mimicked it here. I can take it out if that'd be better.